src/libse/Common/IfoParser.cs

```diff
@@ -39,37 +39,39 @@ public class VtsVobs
             public List<string> GetAllLanguages()
             {
                 var list = new List<string>();
-                for (int i = 0; i < Subtitles.Count; i++)
+                var minCount = Math.Min(Subtitles.Count, Math.Min(SubtitleIDs.Count, SubtitleTypes.Count));
+                for (int i = 0; i < minCount; i++)
                 {
-                    if (i < SubtitleIDs.Count && i < SubtitleTypes.Count)
+                    var ids = SubtitleIDs[i].Split(',');
+                    var types = SubtitleTypes[i].Split(',');
+                    if ((ids.Length == 2 && ids[0].Trim() == ids[1].Trim()) || (ids.Length == 3 && ids[0].Trim() == ids[1].Trim() && ids[1].Trim() == ids[2].Trim()))
                     {
-                        var ids = SubtitleIDs[i].Split(',');
-                        var types = SubtitleTypes[i].Split(',');
-                        if (ids.Length == 2 && ids[0].Trim() == ids[1].Trim() || ids.Length == 3 && ids[0].Trim() == ids[1].Trim() && ids[1].Trim() == ids[2].Trim())
```

Comment:
Bug in this line, the statement should've been grouped with `\( \)` because the "or" performed for ` ids[0].Trim() == ids[1].Trim() || ids.Length == 3`

The correct is `(ids.Length == 2 && ids[0].Trim() == ids[1].Trim()) || (ids.Length == 3 && ids[0].Trim() == ids[1].Trim() && ids[1].Trim() == ids[2].Trim())`